R/reliabilityAnalysisFrequentist.R

```diff
@@ -422,8 +435,7 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
   idxSelected <- which(selected)
   
   if (options[["mcDonaldScale"]] && !is.null(relyFit[["freq"]][["omega.error"]])) {
-    model[["footnote"]] <- gettextf("%sMcDonald's \u03C9 estimation method switched to PFA because the CFA
-                                    did not find a solution. ", model[["footnote"]])
+    model[["footnote"]] <- gettextf("%sMcDonald's %s estimation method switched to PFA because the CFA did not find a solution. ", model[["footnote"]], "\u03C9")
```

Comment:
@juliuspf there's still a change here

Reply:
I thought this was a change that Frans forgot, cause he did change the gettexts with omega in the rest of the files in the same way

Reply:
yeah probably. But I don't want there to be any differences between this file and the one in jasp-desktop, otherwise, the translations files also need to be changed again.